src/global.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
there is a benefit on importing the tokens in `index.ts`? I think the right import was here

Reply:
nope I think we ( I ) removed the wrong line ...